node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -4318,14 +4302,16 @@ mod tests {
     }
 
     #[test]
-    fn standard_gossip_containing_neighbor_node_that_accepts_connections_but_has_no_node_addr_is_rejected(
+    fn standard_gossip_containing_non_transmitting_neighbor_node_that_accepts_connections_but_has_no_node_addr_is_accepted(
```

Comment:
Hmmm, that I'm asking might be because I don't have an enough good nose for this but what do you mean by "non_transmitting_neighbor"...is that "consume only"?

Reply:
No, it means...hmm. Okay: you're Node A, and you have neighbors B and C. Node B sends you Gossip including information about both itself and C. B is the transmitting neighbor, and C is a non-transmitting neighbor.

If B accepts connections but does not transmit its IP address, that's a problem. However, if C accepts connections, it's fine for B not to disclose C's IP address.

I can't come up with a better way to communicate that in the test name; I'll put it in a comment.